apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V72__new_schema_inpatient_claims.sql

```diff
@@ -0,0 +1,960 @@
+                    prcdr_dt13,
+                    icd_prcdr_cd13,
+                    icd_prcdr_vrsn_cd13,
+                    icd_dgns_cd13,
+                    icd_dgns_vrsn_cd13,
+                    clm_poa_ind_sw13,
+                    prcdr_dt14,
+                    icd_prcdr_cd14,
+                    icd_prcdr_vrsn_cd14,
+                    icd_dgns_cd14,
+                    icd_dgns_vrsn_cd14,
+                    clm_poa_ind_sw14,
+                    prcdr_dt15,
+                    icd_prcdr_cd15,
+                    icd_prcdr_vrsn_cd15,
+                    icd_dgns_cd15,
+                    icd_dgns_vrsn_cd15,
+                    clm_poa_ind_sw15,
+                    prcdr_dt16,
+                    icd_prcdr_cd16,
+                    icd_prcdr_vrsn_cd16,
+                    icd_dgns_cd16,
+                    icd_dgns_vrsn_cd16,
+                    clm_poa_ind_sw16,
+                    prcdr_dt17,
+                    icd_prcdr_cd17,
+                    icd_prcdr_vrsn_cd17,
+                    icd_dgns_cd17,
+                    icd_dgns_vrsn_cd17,
+                    clm_poa_ind_sw17,
+                    prcdr_dt18,
+                    icd_prcdr_cd18,
+                    icd_prcdr_vrsn_cd18,
+                    icd_dgns_cd18,
+                    icd_dgns_vrsn_cd18,
+                    clm_poa_ind_sw18,
+                    prcdr_dt19,
+                    icd_prcdr_cd19,
+                    icd_prcdr_vrsn_cd19,
+                    icd_dgns_cd19,
+                    icd_dgns_vrsn_cd19,
+                    clm_poa_ind_sw19,
+                    prcdr_dt20,
+                    icd_prcdr_cd20,
+                    icd_prcdr_vrsn_cd20,
+                    icd_dgns_cd20,
+                    icd_dgns_vrsn_cd20,
+                    clm_poa_ind_sw20,
+                    prcdr_dt21,
+                    icd_prcdr_cd21,
+                    icd_prcdr_vrsn_cd21,
+                    icd_dgns_cd21,
+                    icd_dgns_vrsn_cd21,
+                    clm_poa_ind_sw21,
+                    prcdr_dt22,
+                    icd_prcdr_cd22,
+                    icd_prcdr_vrsn_cd22,
+                    icd_dgns_cd22,
+                    icd_dgns_vrsn_cd22,
+                    clm_poa_ind_sw22,
+                    prcdr_dt23,
+                    icd_prcdr_cd23,
+                    icd_prcdr_vrsn_cd23,
+                    icd_dgns_cd23,
+                    icd_dgns_vrsn_cd23,
+                    clm_poa_ind_sw23,
+                    prcdr_dt24,
+                    icd_prcdr_cd24,
+                    icd_prcdr_vrsn_cd24,
+                    icd_dgns_cd24,
+                    icd_dgns_vrsn_cd24,
+                    clm_poa_ind_sw24,
+                    prcdr_dt25,
+                    icd_prcdr_cd25,
+                    icd_prcdr_vrsn_cd25,
+                    icd_dgns_cd25,
+                    icd_dgns_vrsn_cd25,
+                    clm_poa_ind_sw25
+from
+    public.inpatient_claims;
```

Comment:
I cannot find clm_tot_chrg_amt

Reply:
??? not sure there is such a field...we have: **clm_tot_pps_cptl_amt**

Where did you see/get clm_tot_chrg_amt? I can't find it in either inpatient_claims or inpatient_clim_lines.

Now that being said, it is in the Excel spreadsheet. But it is not in the current schema. Going to check old RIF file(s) to see if it's ever been around (btw, current inpatient_claims table is exact same physical structure....we only ever renamed column names which was a meta-data change 4+ months ago).

Reply:
not sure I understand the comment, given the context was showing the columns (1-25) of icd, etc. So in the case of the clm_tot_chrg_amt, then yes unit and IT(s) would have found that. However, during deployment of the flyway migrations, I typically run a verification of data between original and new tables. I've not included that script in the PR(s) as I did not want to cause people's eyes to glaze over. I'll add them here as an example verification of what happens; I find this type of data verification better than unit or IT(s) since it literally compares orig vs. new table data for EVERY column...I doubt if our unit/ITs even come close to 20% of the data in the INPATIENTS claims.

[verify_inpatient_claims.sql.txt](https://github.com/CMSgov/beneficiary-fhir-data/files/8808090/verify_inpatient_claims.sql.txt)
[verify_inpatient_claim_lines.sql.txt](https://github.com/CMSgov/beneficiary-fhir-data/files/8808093/verify_inpatient_claim_lines.sql.txt)

Reply:
my comment is just that I think we ought to have the code changes done and tests passing against this flyway script before merging this PR even though those code changes are part of another PR.